pkg/cache/external/scheduler_cache.go

```diff
@@ -450,7 +450,7 @@ func (cache *SchedulerCache) StartPodAllocation(podKey string, nodeID string) bo
 }
 
 // return if pod is assumed in cache, avoid nil
-func (cache *SchedulerCache) isAssumedPod(podKey string) bool {
```

Comment:
If we're making this public, we should keep the existing function name, and wrap with an IsAssumedPod() which takes a lock. Otherwise, we risk external callers not locking.